lib/common/db-renderable-content.js

```diff
@@ -191,7 +191,7 @@ function dbRenderableContentServiceFactory(
             i += 1;
         });
 
-        return waterline[self.collectionName].find({name: name})
+        return waterline[self.collectionName].find({name: name, scope: scope})
```

Comment:
Good catch.  The query was structured this way because the Cassandra waterline adapter does not support compound keys and will not support the IN query on the scope key due to that restriction.